components/econet/econet.h

```diff
@@ -43,6 +43,7 @@ struct EconetDatapoint {
   };
   std::string value_string;
   std::vector<uint8_t> value_raw;
+  uint32_t src_adr;
```

Comment:
This doesn't quite belong here. This struct is to hold the value of the datapoint. I think the source address belongs to EconetDatapointListener. That should also make future work to have address per object easier.

Reply:
Agree. 

If we do want to jump there, this will be a much much bigger change as we would need to track data points values by source address and object name to determine if it has changed and likely need to update all the individual types as well.

We will have to deal with the from any source address case and the specific source address changes.

This also compounds to handling by datapoint (which for this would be a yes). And then how that interacts with request_mods...

Reply:
I take that back... we probably eventually do want to do some of the things above, but its not a must do right now. Changed this to be in EconetDatapointListener instead.